### ISSUE TITLE
Add option to act as Rails console

### ISSUE THREAD

**Issue by walter** [jacott/emacs-pry#7]
I may have overlooked this some place, but I'm wondering if there is a way for emacs-pry to act as a console for a Rails application.

**Comment by Trevoke** [comment]
Well, there is pry-rails already -- it would be very neat if emacs-pry supported pry-rails. In particular because on my team there is now an emacs user who is now bothered by pry because it doesn't play well with emacs (which I'm sure was the source for this in the first place).

**Comment by walter** [comment]
Yeah, I'm using pry-rails and simply calling it from within a shell-mode buffer as that works as expected. I'll update here if I find a way that people load environment from a Rails app.
